python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -16,85 +16,117 @@ tables:
     datatype: long
     description: Unique id. Unique ObjectID
     ivoa:ucd: meta.id;src;meta.main
+    tap:principal: 1
+    tap:column_index: 1
   - name: coord_dec
     "@id": "#Object.coord_dec"
     datatype: double
     description: Fiducial ICRS Declination of centroid used for database indexing
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec;meta.main
+    tap:principal: 1
+    tap:column_index: 4
   - name: coord_ra
     "@id": "#Object.coord_ra"
     datatype: double
     description: Fiducial ICRS Right Ascension of centroid used for database indexing
     fits:tunit: deg
     ivoa:ucd: pos.eq.ra;meta.main
+    tap:principal: 1
+    tap:column_index: 3
   - name: deblend_nChild
     "@id": "#Object.deblend_nChild"
     datatype: int
     description: Number of children this object has (defaults to 0)
     fits:tunit:
+    tap:principal: 0
```

Comment:
Not in this PR, but it's not really that this column has a "default"; it's that isolated objects and deblended children do not have any children themselves.